OpenEdXMobile/src/main/java/org/edx/mobile/http/notifications/EdxErrorState.kt

```diff
@@ -54,6 +54,11 @@ class EdxErrorState @JvmOverloads constructor(
                 layout.action.iconPadding =
                     context.resources.getDimension(R.dimen.container_padding).toInt()
             }
+            State.UPDATE_APP -> {
+                layout.icon.setImageDrawable(R.drawable.ic_update_app)
+                layout.errorText.text = context.getText(R.string.app_version_unsupported)
```

Comment:
- color should be `primaryDarkColor` for the error text.
- There is no Full stop at the end of the text for `app_version_unsupported`.

Reply:
The colour for [empty state](https://www.figma.com/file/Vaw3lfBXmOhTLKFnBedqE1/App-Navigation-Improvements?node-id=4669%3A17826&t=iMgV8ydLA3y57Zmf-0) is `neutralBlackT(#000000)` and  `primaryDarkColor(#002121)` for the [upgrade state](https://www.figma.com/file/Vaw3lfBXmOhTLKFnBedqE1/App-Navigation-Improvements?node-id=4669%3A17827&t=iMgV8ydLA3y57Zmf-0). Please ask Hilary if it is intentional.